DESCRIPTION

```diff
@@ -1,18 +1,36 @@
 Type: Package
 Package: mlfit
-Title: Implementation of algorithms that extend IPF to nested structures
+Title: Implementation of algorithms that extend Iterative Proportional Fitting to nested structures
 Version: 0.4.0
 Date: 2021-04-26
 Authors@R: 
-     c(person(given = "Kirill", 
-              family = "Müller", 
-              role = c("aut","cre"), 
-              email = "krlmlr+r@mailbox.org"),
-       person(given = "Amarin",
-              family = "Siripanich", 
-              role = c("ctb"), 
-              email = "amarin.siri@gmail.com"))
-Description: The IPF algorithm operates on count data. This package offers
+    c(
+        person(
+            given = "Kirill",
+            family = "Müller",
+            role = c("aut", "cre"),
+            email = "krlmlr+r@mailbox.org"
+        ),
+        person(
+            given = "Kay",
+            family = "Axhausen",
+            role = "ths",
+            email = "axhausen@ivt.baug.ethz.ch"
```

Comment:
I'm not sure we need e-mails for anyone but the maintainer.

Reply:
Removed all emails, except for mine.